template-downtime.yaml

```diff
@@ -14,11 +14,11 @@
   Severity: <TEXT>
   # Start date and time of the donwtime in the following format:
   # <MONTH> <DAY>, <YEAR> <TIME> <TIME ZONE>
-  # e.g. Mar 28, 2017 22:00 UTC
+  # e.g. Mar 28, 2017 22:00 -0500
```

Comment:
Just to note here: you are changing the example timezone from UTC to (presumably) CDT.  Not sure which is more common for sysadmins, eg, based on server log timestamps.